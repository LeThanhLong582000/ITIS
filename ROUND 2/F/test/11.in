100
50058637 25374623 64682729
67340867 12786761 51039616
61279695 19019370 31355945
28272828 17434859 19046506
64577979 99600322 60585194
68883963 89680023 29914887
77572235 12742349 76705136
85781262 14173944 70497438
93051876 47603859 17199100
11887538 17610226 44027711
31432913 55323088 81290497
48338195 40206840 66591929
3513984 52258315 5589740
16257372 28714792 52940541
63692849 11917200 38638579
76607485 15272028 44715825
15979652 86387061 28155357
72856609 14331822 30296583
99725970 93258871 55384731
78000206 21791729 6136394
75984451 7258052 77281829
56485160 96282130 92712916
56895790 82514689 18430027
36428599 94874669 71600652
82464509 57093522 61633371
20945534 6147500 15534785
83034944 47498688 92276649
6952485 69389896 57197462
84730011 12628182 23986669
80328297 66300091 37807633
34823412 88159773 3361500
88702423 53633854 91002574
37991205 53018333 22306835
47935090 7426381 9064290
17316660 80974310 32134508
57345374 9421805 74802433
66272762 61495077 48476552
10395812 45389297 41830047
39805754 33673147 15875072
35725779 89575209 57071168
53558528 31820999 90405457
66161105 1465868 4778474
13038047 77860150 48975255
38813580 20821571 2370754
70330405 86020530 21177441
68140803 84400664 91071757
14611057 96365330 13452483
29923576 33807361 45883183
46775899 89781786 2750011
10651846 20229898 29785645
31831545 89970720 32426726
4039142 38318199 95471778
25293796 66862720 30508123
46551238 39339536 42240179
19012702 8342991 83292405
34778704 85879231 13256942
34136320 59566601 28287074
7816407 10950898 7144889
25526210 10383156 18178212
75976871 74542136 2635809
14319624 22978612 67833098
49712587 39273047 44778163
96696985 73946012 41702747
65981333 6279843 79723516
77314106 63491693 5521364
23253826 45345137 66801929
69368941 56256691 24210393
62031191 80803300 95281207
73712063 36037975 86037945
47728015 18547008 81190653
88258584 16371320 75891446
30441229 58964487 21829525
17215860 72622112 30177585
57659261 62448841 11745069
83303876 19317858 33308698
75779491 89027600 89999181
71397638 58841485 55177747
85458321 27349362 1158054
52299320 88074378 76578774
13341549 97752162 20389419
96004086 5064828 34804086
49114353 2150453 56649444
98282314 95013366 85099297
83466677 81392294 61804734
92238307 73299356 3782503
23038967 10150243 95324960
1580558 28587847 25810006
92173617 64684966 36799251
15168443 41523765 63827772
76726855 22992178 72856705
1151327 28122365 23916236
83048366 99337177 81874110
86888051 34101510 8628250
63349892 96985413 61834997
78816887 16598284 52406954
93333789 41677459 2446488
33780759 1085342 50405662
78700881 88901299 62918142
61192574 46589611 83741428
83635243 83618035 59336221
