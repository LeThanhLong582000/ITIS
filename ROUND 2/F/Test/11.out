44683372
51039617
31355946
14942238
12781426
4559414
70767511
70497439
17199101
19152512
28700161
37361642
0
20241561
38638580
44715826
0
30296584
30925915
6136395
73004114
26457973
0
6577291
43502179
15166410
63906453
0
23986670
25917920
0
63035572
3639854
9064291
0
61363001
26627119
3418281
11003840
1610869
56071493
4778475
0
2370755
2743658
37405948
0
20999699
0
10103797
0
30596361
0
24725941
46981058
0
1428397
2005199
16660633
2035272
29587055
27608852
32226860
69712503
5521365
22355309
18661322
38254549
61856017
55185830
73889355
0
0
3477745
33308699
38375536
33866950
1158055
20401858
0
34804087
51806672
44184123
31939559
3782504
54106842
0
32143951
18736225
63295691
0
32792650
8628251
14099738
52406955
2446489
41550540
26358862
49172196
29676715
